docker.sh

```diff
@@ -37,10 +51,19 @@ build_tag(){
 # and will alias it with "latest" tag.
 build(){
   new_tag=$(build_tag)
+  case $ARCH in
+    riscv | riscv64)
+      dockerfile=Dockerfile.riscv64
+      ;;
+    *)
+      dockerfile=Dockerfile
+      ;;
+  esac
+  echo "docker build"
   docker build -t "$new_tag" \
         --build-arg GIT_BRANCH="${GIT_BRANCH}" \
         --build-arg GIT_COMMIT="${GIT_COMMIT}" \
-        -f Dockerfile .
+        -f $dockerfile .
```

Comment:
What about using `docker buildx build --platform`? (https://docs.docker.com/build/building/multi-platform/)

If it is too complicated, I'm fine to discard this suggestion ;-)

Reply:
Not using buildx because  
unlike x86/ARM CI, currently there is no RISC-V Buildkite agent platform.  
As a workaround, QEMU is used to run RISC-V CI on x86 Buildkite agent platform.